src/app/portal/features/account/account.component.html

```diff
@@ -0,0 +1,81 @@
+<h1 class="text-center">
+  <span class="text-3xl">User Account</span><br />
+  <span class="text-3xl">{{ auth.fullName() }}</span>
+</h1>
```

Comment:
Maybe would be better to use two headings of different levels or to use different colors for the title and the name.